src/base/utils/misc.cpp

```diff
@@ -588,24 +589,11 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
     // QProcess::startDetached() explorer.exe still shows the default path. In
     // this case QProcess::startDetached() probably puts its own quotes around ours.
 
-    STARTUPINFO startupInfo;
-    ::ZeroMemory(&startupInfo, sizeof(startupInfo));
-    startupInfo.cb = sizeof(startupInfo);
-
-    PROCESS_INFORMATION processInfo;
-    ::ZeroMemory(&processInfo, sizeof(processInfo));
-
-    QString cmd = QString("explorer.exe /select,\"%1\"").arg(Utils::Fs::toNativePath(absolutePath));
-    LPWSTR lpCmd = new WCHAR[cmd.size() + 1];
-    cmd.toWCharArray(lpCmd);
-    lpCmd[cmd.size()] = 0;
-
-    bool ret = ::CreateProcessW(NULL, lpCmd, NULL, NULL, FALSE, 0, NULL, NULL, &startupInfo, &processInfo);
-    delete [] lpCmd;
-
-    if (ret) {
-        ::CloseHandle(processInfo.hProcess);
-        ::CloseHandle(processInfo.hThread);
+    CoInitializeEx(NULL, COINIT_MULTITHREADED);
+    ITEMIDLIST *pidl = ILCreateFromPathW(reinterpret_cast<PCTSTR>(Utils::Fs::toNativePath(path).utf16()));
+    if (pidl) {
+        SHOpenFolderAndSelectItems(pidl, 0, nullptr, 0);
+        ILFree(pidl);
     }
```

Comment:
>You have mixed usage of nullptr and NULL. Since this is WINAPI should we always use NULL. @glassez thoughts?

`NULL` is old C macro used for compatibility purposes. C++11 has exact type for null pointers and keyword `nullptr` which is constant of this type. So I think we should use `nullptr` for all null pointers in new code.